include/zrepl_prot.h

```diff
@@ -76,10 +76,12 @@ typedef enum zvol_op_status zvol_op_status_t;
 struct zvol_io_hdr {
 	uint16_t	version;
 	zvol_op_code_t	opcode;
-	zvol_op_status_t status;
 	uint64_t	io_seq;
 	uint64_t	offset;
 	uint64_t	len;
+	uint64_t	checkpointed_io_seq;
+	uint8_t 	flags;
```

Comment:
what is this used for? Also if there are new flags introduced to protocol they should be defined somewhere in this header file.

Reply:
This flag would be used to differentiate if read/write op meant for replication or rebuild. We do couple of things differently for rebuild from replication. Yes flags would go to "include/zrepl_port.h" file.

Reply:
I think it would make sense to define the flag now when we introduce the new field. Something like:

```
#define IO_FLAG_REBUILD   0x1
```